src/main/scripts/add_bam_pair_xenocp.py

```diff
@@ -109,19 +109,25 @@ def add_bam_pair_stp(raptr, sample, target, project, subproject, **kwargs):
     # Get genome for old bam
     query = """select genome_id from bam where bam_id = %s;"""
     genome_id = raptr.fetch_item_or_fail(query, (bam_id,))
+
+    # Get anls type named xenocp
+    query = """select anls_type_id from raptr.anls_type where name='%s'"""
+    anls_type_id = raptr.fetch_item_or_fail(query, ('xenocp',))
+
     # Add a new bam
-    query = """insert into bam (bam_id, bam_tpl_id, status, notes, genome_id)
-            values (nextval('blt_id_seq'), %s, 'Normal', NULL, %s) returning bam_id;"""
-    bam_id_xenocp = raptr.fetch_item_or_fail(query, (bam_tpl_id_xenocp, genome_id))
+    query = """insert into bam (bam_id, bam_tpl_id, status, notes, genome_id, anls_type_id, source_bam_id)
+            values (nextval('blt_id_seq'), %s, 'Normal', NULL, %s, %s, %s) returning bam_id;"""
```

Comment:
Should this nextval('blt_id_seq') be in here, or should we omit now?  Can't remember the order of the updates.

Reply:
My plan was to do those changes separately, though since this is a fairly small change, it could be bundled together if you think that is a better idea.

Reply:
OK, as long as it was intentional.  I just didn't remember the order.